Kickstarter-iOS/Views/FundingGraphView.swift

```diff
@@ -255,30 +255,6 @@ private func slope(ofLine line: Line) -> CGFloat {
 private func dateToDayNumber(launchDate: TimeInterval,
```

Comment:
@luoser turns out your `dateToDayNumber` and `totalNumberOfDays` do the same thing, so i removed one, and the definition of the function could be simplified.